pkg/interop/types.go

```diff
@@ -35,3 +37,40 @@ type PublicKey []byte
 // Interface represents interop interface type which is needed for
 // transparent handling of VM-internal types (e.g. storage.Context).
 type Interface interface{}
+
+// Equals compares Hash160 with the provided stackitem using EQUAL opcode.
```

Comment:
Maybe worth mentioning that it only works for int, bool, string or []byte `b`, because structure or other array will fail on `.(string)` conversion.